src/App.css

```diff
@@ -1,6 +1,7 @@
 @import "https://dreamui-demo.netlify.app/dreamui-v1.css";
 @import "./components/Home/Home.css";
 @import "./components/products/Products.css";
+@import "./components/Cart/cart.css";
```

Comment:
can use barrel imports here

Reply:
What are barrel imports?

Reply:
> What are barrel imports?

create a index.js file and export all the components from one single file